### PR TITLE
Add listening for ws-client-connected and handle it clearing the comp…

### PR DESCRIPTION
**Pull request by rodrigopavezi**
…onent list

### CODE REVIEW

**Comment by rodrigopavezi** [comment]
[Delete components after checking stable/unstable status [3]](https://trello.com/c/4IeMuw4Z/4672-delete-components-after-checking-stable-unstable-status-3)

File: src/components/components-controller.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ function updateAllComponents() {
 
   if (!twitter.credentialsExist()) {return logger.file("Credentials do not exist - can not update components");}
 
-  twitter.closeStream();
+  twitter.closeAllStreams();
```
Comment (jungeunyoon) [inline src/components/components-controller.js]: @rodrigopavezi good catch

File: src/messaging/update/update.js [hunk 2]
```diff
@@ -23,5 +23,8 @@ module.exports = {
   },
   processAll() {
     componentsController.updateAllComponents();
+  },
+  clear() {
+    componentsController.clearComponents();
```
Comment (jungeunyoon) [inline src/messaging/update/update.js]: @rodrigopavezi We don't have to close all streams here?

Comment (rodrigopavezi) [reply src/messaging/update/update.js]: @jungeunyoon good catch, completely forgot about that. cheers

**Pull request merged**